### PR TITLE
Pipes advanced

### PR DESCRIPTION
**Pull request by nk1tz**
Done

### CODE REVIEW

File: poem-lines.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+There are 16 lines in poem.txt
```
Comment (ziad-saab) [inline poem-lines.txt]: How did you get to this result? Did you run a command or count the lines manually?

File: poem-words.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Tthere are 60 words in poem.txt
```
Comment (ziad-saab) [inline poem-words.txt]: How did you get to this result? Did you count the words manually?

File: poem.txt [hunk 3]
```diff
@@ -7,7 +7,7 @@ Who stands by you.
 A friend is someone
 Who lifts your spirits.
 A friend is someone
-who encourages you.
+Who encourages you.
```
Comment (ziad-saab) [inline poem.txt]: :open_mouth: you changed my awesome poem!

File: poem-lines.txt (hunk 1 above)
Comment (nk1tz) [inline poem-lines.txt]: I ran a command to count then I went to nano and edited the result to make it look better.

**Pull request closed without merging**